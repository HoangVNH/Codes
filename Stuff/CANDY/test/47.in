10
99
VDDXXVVDDXVXXVXXXVXDDXVVDVXXVXDVVXDVDXDVDDDDDDXXXDVDXVVDDXDDXVDVXXXXVVVDXXVXVDDXDVVVXVDDDVDDDDVVXVX
80
DVVVDDXDXVDVVXVDXXVXXVVDDXVDXVXDXDVVDVDDDVXXVXVXDVXVDVXDDXVVDDXDDVXXXDDVXVVDDDXV
50
DVDXDDXDVVDDXDDDDVDVVDDDXVDXVDVVXXDXXDVVXVXVDDVDVD
60
DXDDXXVVXDXVDDXXVVVXXVDDXDVXXVDXVDXXXXDXXDXDDVVDXVXVXVDXDDVX
80
XVVDDVDXXDDXVXXVDXVXXXXVXXDXVVXVVDXVVXDVXVXDDVVDVXVXVXDDDVDVDXXDXXVXDDVVVDXVDVDD
88
XDDXXVXXVVDDVDVVVDXDDDVXVVDXDDDXXVXVXDDVXVXDDDVVXVDXXXXVXVXXDDDVVVDVXXXVVDDXDXDVDDXXXDXV
72
XXVXXXDVVVXVVDXDXDVXDVDXVVDVXXVDXVVDDDVVXXDDXDXDVDVDXVXDVDVDVDDDXVXXDDDX
80
VXVVXVDVVVXVXXXVDVXDDDVDDDXDDVXVVDXXVVDXXDVDXDVXDVDXXXDDDXXDVVVVVVXVVVVXXXVVVVXX
83
VDXDXVVVXXVXVXDVVVVVDDXVVXXXVXXXXDXDVXDXVDVXXVVXDVVXDXDXXDXXXVVDDVXXXDVDXXDDDDDXVXV
71
XXXXXVXXVXXXDVVXDXXDVDVDVVXVXXXVVDXDVDXDDDDDXVXDVDDDVVXDVVXVVVVDDVVVVVD
