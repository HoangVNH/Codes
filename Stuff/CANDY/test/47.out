98
78
49
59
78
87
71
79
82
69
